UI/src/view/linear_asset/speedLimitStyle.js

```diff
@@ -188,7 +208,7 @@
         return GeometryUtils.offsetBySideCode(applicationModel.zoom.level, speedLimit);
       };
       var speedLimitsWithAdjustments = _.map(speedLimitsWithType, offsetBySideCode);
-      var speedLimitsSplitAt70kmh = _.groupBy(speedLimitsWithAdjustments, function(speedLimit) { return speedLimit.value >= 70; });
+      var speedLimitsSplitAt70kmh = _.groupBy(speedLimitsWithAdjustments, function(speedLimit) { return !_.isUndefined(speedLimit.value) ? speedLimit.value.value >= 70 : false; });
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code